### PR TITLE
INT-B-20472

### PR DESCRIPTION
**Pull request by loganwc**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

### How to test

1. Login to milmovelocal
2. Create a new move
3. Before submitting move click the 'Cancel Move' button
4. Check database for the canceled move and it's shipments
5. Create a new move
6. Submit move to SC 
7. Cancel Move button should be gone

### CODE REVIEW

File: src/pages/MyMove/Home/MoveHome.jsx [hunk 1]
```diff
@@ -523,7 +562,7 @@ const MoveHome = ({ serviceMemberMoves, isProfileComplete, serviceMember, signed
                 {!hasSubmittedMove() && (
                   <Step
                     complete={hasOrdersAndUpload()}
-                    completedHeaderText="Orders uploaded"
+                    completedHeaderText="window.location.reloadOrders uploaded"
```
Comment (paulstonebraker) [inline src/pages/MyMove/Home/MoveHome.jsx:565]: whats this

Comment (loganwc) [reply src/pages/MyMove/Home/MoveHome.jsx:565]: I've never seen this before

Comment (loganwc) [reply src/pages/MyMove/Home/MoveHome.jsx]: undid that change

**Comment by pambecker** [comment]
'cancelled' should be spelled with 1 'l' to match the rest of the UI (canceled).
image

**Comment by pambecker** [comment]
Maybe something to ask Terri, do we want to redirect the customer back to the landing page after cancel?  After I click yes in the modal, it just stays in the move but I have no way to tell if the move was canceled until I go back to the landing page.  Also, we will need a success message to display after the submit.

**Comment by loganwc** [comment]
> Maybe something to ask Terri, do we want to redirect the customer back to the landing page after cancel? After I click yes in the modal, it just stays in the move but I have no way to tell if the move was canceled until I go back to the landing page. Also, we will need a success message to display after the submit.

There was a toast confirming that the move was canceled but it wouldn't display because I was reloading the page. I removed that and made the other changes you requested 👍

File: pkg/handlers/internalapi/office_test.go [hunk 2]
```diff
@@ -138,103 +138,67 @@ func (suite *HandlerSuite) TestApproveMoveHandlerForbidden() {
...
-	handler := CancelMoveHandler{handlerConfig, moveRouter}
-	response := handler.Handle(params)
+		// And: a move is canceled
+		handlerConfig := suite.HandlerConfig()
+		handler := CancelMoveHandler{handlerConfig, moveRouter}
+		response := handler.Handle(params)
 
-	// Then: response is Forbidden
-	suite.Assertions.IsType(&officeop.CancelMoveForbidden{}, response)
+		// Then: expect a 200 status code
```
Comment (paulstonebraker) [inline pkg/handlers/internalapi/office_test.go:199]: Update comments for this section

**Review by paulstonebraker — CHANGES_REQUESTED** [review]
Works as intended, only change requested is updating comments. I have a couple more thoughts that I will share as non blocking comments

**Comment by paulstonebraker** [comment]
Two thoughts:

1. in the handler, do we want to check if the logged in user's ID is the same as the `Move.Order.ServiceMemberID` and return an error if not?
2. customer viewing a canceled move looks kinda weird
image

image

image

**Review by JonSpight — APPROVED** [review]
Other than the updated comment, pointed out by Paul. I am good too.

**Review by joeydoyecaci — APPROVED** [review]
No issue with cancelling other than the PPM glitch and the "d" screen issue.

**Review by paulstonebraker — APPROVED** [review]

**Pull request merged**